test/steps.go

```diff
@@ -104,11 +105,16 @@ func Steps(suite *godog.Suite, state *ScenarioState) {
 	suite.Step(`^Git Town is (?:now|still) aware of this branch hierarchy$`, func(input *messages.PickleStepArgument_PickleTable) error {
 		table := DataTable{}
 		table.AddRow("BRANCH", "PARENT")
-		for _, row := range input.Rows[1:] {
-			branch := row.Cells[0].Value
-			state.gitEnv.DevRepo.Config.Reload()
-			parentBranch := state.gitEnv.DevRepo.Config.GetParentBranch(branch)
-			table.AddRow(branch, parentBranch)
+		state.gitEnv.DevRepo.Config.Reload()
+		// Table sorted by child branch name
+		parentBranchMap := state.gitEnv.DevRepo.Config.GetParentBranchMap() 
+		childBranches := make([]string, 0, len(parentBranchMap))
+		for child := range parentBranchMap {
+			childBranches = append(childBranches, child)
+		}
```

Comment:
Can this be simplified to:

```suggestion
		childBranches = append(childBranches, parentBranchMap...)
```

Reply:
Doesn't appear so.

Get the error:
```
test/steps.go:112:25: cannot use parentBranchMap (type map[string]string) as type []string in append
```

Was thinking if that was possible why that isn't used as the defacto way to get a map's keys